distribution/src/main/packaging/scripts/postinst

```diff
@@ -61,9 +61,9 @@ if [ "x$IS_UPGRADE" != "xtrue" ]; then
 
     elif command -v chkconfig >/dev/null; then
```

Comment:
> running `sudo command -p chkconfig` doesn't find `command`!

`command` is a shell built-in instead of an actual command (another reason I don't really like it), personally I'd rather hardcode paths or use something like `/usr/bin/which chkconfig`